### PR TITLE
Create goentri.com.owner.json

### PR DESCRIPTION
**Pull request by evelyndearaujo-goentri**

### CODE REVIEW

**Review by wdbaker-godaddy — COMMENTED** [review]
Please consider using variable %ip%.  I believe some DNS Providers have trouble with single character variables.  Thx

File: goentri.com.owner.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  "description": "Allows user to point domain to the website",
+  "syncPubKeyDomain": "goentri.com",
+  "syncRedirectDomain": "api.goentri.com, goentri.com, entri.com, app.entri.com,app.goentri.com",
+  "variableDescription": "a is  IP record",
+  "sharedProviderName": true,
+  "records": [
+    {
+      "type": "A",
+      "host": "@",
+      "pointsTo": "%a%",
```
Comment (vkasam-godaddy) [inline goentri.com.owner.json:17]: As suggested by wdbaker-godaddy,  can we please consider using variable %ip% here? We believe some DNS providers have trouble with single character variables. Thanks!

Comment (evelyndearaujo-goentri) [reply goentri.com.owner.json]: Thanks for the suggestion @vkasam-godaddy 
I updated variable name.

**Review by wdbaker-godaddy — APPROVED** [review]
Approved

**Pull request merged**